keyboards/system76/launch_1/info.json

```
@@ -0,0 +1,94 @@
+                { "label": "5", "x": 5, "y": 1 },
+                { "label": "6", "x": 6, "y": 1 },
+                { "label": "7", "x": 7, "y": 1 },
+                { "label": "8", "x": 8, "y": 1 },
+                { "label": "9", "x": 9, "y": 1 },
+                { "label": "0", "x": 10, "y": 1 },
+                { "label": "-", "x": 11, "y": 1 },
+                { "label": "=", "x": 12, "y": 1 },
+                { "label": "Bksp", "x": 13, "y": 1, "w": 1.5 },
+                { "label": "PgUp", "x": 14.75, "y": 1 },
+                { "label": "Tab", "x": 0, "y": 2, "w": 1.5 },
+                { "label": "Q", "x": 1.5, "y": 2 },
+                { "label": "W", "x": 2.5, "y": 2 },
+                { "label": "E", "x": 3.5, "y": 2 },
+                { "label": "R", "x": 4.5, "y": 2 },
+                { "label": "T", "x": 5.5, "y": 2 },
+                { "label": "Y", "x": 6.5, "y": 2 },
+                { "label": "U", "x": 7.5, "y": 2 },
+                { "label": "I", "x": 8.5, "y": 2 },
+                { "label": "O", "x": 9.5, "y": 2 },
+                { "label": "P", "x": 10.5, "y": 2 },
+                { "label": "[", "x": 11.5, "y": 2 },
+                { "label": "]", "x": 12.5, "y": 2 },
+                { "label": "\\", "x": 13.5, "y": 2 },
+                { "label": "PgDn", "x": 14.75, "y": 2 },
+                { "label": "Caps", "x": 0.25, "y": 3, "w": 1.5 },
+                { "label": "A", "x": 1.75, "y": 3 },
+                { "label": "S", "x": 2.75, "y": 3 },
+                { "label": "D", "x": 3.75, "y": 3 },
+                { "label": "F", "x": 4.75, "y": 3 },
+                { "label": "G", "x": 5.75, "y": 3 },
+                { "label": "H", "x": 6.75, "y": 3 },
+                { "label": "J", "x": 7.75, "y": 3 },
+                { "label": "K", "x": 8.75, "y": 3 },
+                { "label": "L", "x": 9.75, "y": 3 },
+                { "label": ";", "x": 10.75, "y": 3 },
+                { "label": "'", "x": 11.75, "y": 3 },
+                { "label": "Enter", "x": 12.75, "y": 3, "w": 1.5 },
+                { "label": "End", "x": 14.75, "y": 3 },
+                { "label": "LShift", "x": 0.25, "y": 4, "w": 2 },
+                { "label": "Z", "x": 2.25, "y": 4 },
+                { "label": "X", "x": 3.25, "y": 4 },
+                { "label": "C", "x": 4.25, "y": 4 },
+                { "label": "V", "x": 5.25, "y": 4 },
+                { "label": "B", "x": 6.25, "y": 4 },
+                { "label": "N", "x": 7.25, "y": 4 },
+                { "label": "M", "x": 8.25, "y": 4 },
+                { "label": ",", "x": 9.25, "y": 4 },
+                { "label": ".", "x": 10.25, "y": 4 },
+                { "label": "/", "x": 11.25, "y": 4 },
+                { "label": "RShift", "x": 12.25, "y": 4, "w": 1.5 },
+                { "label": "Up", "x": 13.75, "y": 4 },
+                { "label": "LCtrl", "x": 0.25, "y": 5, "w": 1.5 },
+                { "label": "LAlt", "x": 1.25, "y": 5 },
```

Fixes a key overlap.

```suggestion
                { "label": "LAlt", "x": 1.75, "y": 5 },
```

---

Thanks. Fixed.